analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ObjectsShouldNotBeDisposedMoreThanOnce.cs

```diff
@@ -151,4 +166,33 @@ public static void LoopWithBreak(System.Collections.Generic.IEnumerable<string>
             }
         }
     }
+
+    public static void Loop(System.Collections.Generic.IEnumerable<string> list, bool condition, IInterface1 instance1)
```

Comment:
Easier way to have something that can be just iterated
```suggestion
    public static void Loop(string[] list, bool condition, IInterface1 instance1)
```